backend/app/services/inbox/auto_process_handler.py

```
@@ -0,0 +1,337 @@
+            f"message_id={event.message_id}, queue_id={event.queue_id}"
+        )
+
+        try:
+            with get_db_session() as db:
+                # Load WorkQueue
+                work_queue = (
+                    db.query(Kind)
+                    .filter(
+                        Kind.id == event.queue_id,
+                        Kind.kind == "WorkQueue",
+                        Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)